src/overlays/actors/ovl_Bg_Ice_Shelter/z_bg_ice_shelter.c

```diff
@@ -158,6 +160,8 @@ void BgIceShelter_Init(Actor* thisx, PlayState* play) {
     }
 
     if (type == RED_ICE_KING_ZORA) {
+        static Vec3f kzIceScale = { 0.18f, 0.27f, 0.24f };
+
```

Comment:
obviously not your change (I believe this is my mistake from multiple years ago), but can you rename this to `sKingZoraRedIceScale` 
or we can keep it as `Kz`, but its less of a common acronym and I think and doesnt hurt to be descriptive in this case